components/BattleScreen/Menu/Menu.js

```diff
@@ -30,20 +33,49 @@ export default function Menu({ onAttack, disabled }) {
 			) : (
 				<MenuOverviewBox>
 					<MenuOverviewBoxLeft>
-						<p>Klicke auf Kampf um anzugreifen</p>
+						<p>Was soll Glurak tun?</p>
 					</MenuOverviewBoxLeft>
 					<MenuOverviewBoxRight>
-						<MenuButtonFight onClick={handleAttackClick} disabled={disabled}>
+						<MenuButtonFight
+							onMouseEnter={() => {
+								playSound("menuSound");
+								setHoveredButton(1);
+							}}
+							onMouseLeave={() => {
+								stopSound("menuSound");
+								setHoveredButton(null);
+							}}
+							onClick={handleAttackClick}
+							disabled={disabled}
+						>
+							{hoveredButton === 1 && <CursorImage src="/sprites/cursor.png" />}
```

Comment:
This is only a question (no need to change things): Did you consider/try to change the cursor via CSS? It seems to me that `hoveredButton` is only used to switch the cursor image. If this is possible with CSS (using the :hover pseudo class and [custom cursors](https://blog.logrocket.com/creating-custom-mouse-cursor-css/)) only it would be more effective/faster to do that.

Reply:
yes, i considered that in advance as well and also had a version like that, however i liked it better that way.
Especially the use of `<image>` would have meant a few lines of code more, with `<img>` could have been considered, but there has me the yellow underlined line too much triggert :-D

To make the code cleaner and clearer I could possibly make another `handleHover` function, I'm thinking about it, thought about adding a "clean up" user story soon. :-)

Reply:
The point of using CSS would be to make the code faster and more reactive. Because CSS handling is built into the browser, no JavaScript would be executed and no rerendering of React components would occur. But as I said - no need to change things, because it seems to work well!